controllers/cluster_controller.go

```diff
@@ -47,6 +46,7 @@ const (
 	deleteRequeueAfter = 5 * time.Second
 )
 
+// +kubebuilder:rbac:groups=cluster.x-k8s.io,resources=clusters;clusters/status,verbs=get;list;watch;create;update;patch;delete
```

Comment:
To make sure we have properly formatted godoc, and the kubebuilder tags don't leak into it, could you please do it like this (make sure there is a blank line in between the kubebuilder tags and the godoc for the type)? This comment also applies to all the changes below.
```go
// +kubebuilder:rbac:groups=cluster.x-k8s.io,resources=clusters;clusters/status,verbs=get;list;watch;create;update;patch;delete

// ClusterReconciler reconciles a Cluster object
type ClusterReconciler struct {
```

Reply:
done